/**:
  ros__parameters:
    # OPTMIZER params:

    # Model Parameters:
    m: 230.0 # Massa
    Iz: 140.0 # Momento d'inerzia
    Lf: 0.800 # Distanza CoG-Asse anteriore
    Lr: 0.735 # Distanza CoG-Asse posteriore
    Dr: 2800.0  # Costante di Pacejka D (scalamento forza)
    Df: 2400.0  # Costante di Pacejka D (scalamento forza)
    Cr: 1.6 # Costante di Pacejka C (picco forza)
    Cf: 1.6 # Costante di Pacejka C (picco forza)
    Br: 10.1507 # Costante di Pacejka B (forma curva forza)
    Bf: 10.8529 # Costante di Pacejka B (forma curva forza)
    u_r: 0.04 # Resistenza al rotolamento (% al peso veicolo)
    gravity: 9.81
    Cd: 1.13583 # Coeff di attrito aerodinamico
    rho: 1.255 # Densita aria
    Ar: 1.04 # Area frontale
    dRd: 5.0 # Penalizza variazioni di sterzo (Alto = svolte morbide poco zigzag)
    dRa: 4.0 # Penalizza variazioni di accelerazione (Alto = accelerazioni fluide)
    q_slip: 5.0 # Penalizza slittamento (Alto = stabile)
    p_long: 5.0 # Peso su accelerazione longitudinale (Alto = accelerazioni graduali)
    q_n: 0.5 # Penalizza deviazione laterale (Alto = resto sulla midline)
    q_mu: 0.5 # Penalizza deviazione del coefficiente di attrito
    lambda: 0.01 # Determina quanto prudente deve essere la guida (Basso = molto)
    q_s: 0.005 # Pesa progresso lungo la traiettoria. Valore alto veloce ma sporco, basso lento ma pulito

    # Config
    curvatureStride: 10 # 10 # Discretization of middle trajectory's curvature
    horizonLength: 0  # (if > 0) it sets the horizon length of the NLOP --> curvatureStride won't be used
    debugging: false  # debug flag (if true debug function is called)

    Paths:
      curvature: data/curvature.csv
      save: data/